e2e/benchmarks/benchmark_util.js

```diff
@@ -553,6 +557,40 @@ async function setEnvFlags(flagConfig) {
   }
 }
 
+/**
+ * Set flags from URL. URL should be in the format:
+ * ?tfjsflags=FLAG1:1,FLAG2:true.
+ */
+function setEnvFlagsFromUrlParameters() {
+  const TENSORFLOWJS_FLAGS_PREFIX = 'tfjsflags';
+  const urlParams = tf.env().getQueryParams(location.search);
+  if (TENSORFLOWJS_FLAGS_PREFIX in urlParams) {
+    const keyValues = urlParams[TENSORFLOWJS_FLAGS_PREFIX].split(',');
+    keyValues.forEach(keyValue => {
+      const [key, value] = keyValue.split(':');
+      try {
+        tf.env().set(key, parseValue(value));
+      } catch (err) {
+        console.error(err);
+      }
+    });
+  }
+}
+
+/**
+ * Converted a URL parameter to a typed value, such a boolean, number, string.
+ */
+function parseValue(value) {
```

Comment:
Can we export this function from environment.ts?

Reply:
This feature has been supportted by tfjs-core. We add this because e2e is an exception(e2e supports both url flags and tunable flags). For most app, this logic already works. I didn't see any other tfjs app requires this API.

Reply:
I just hope to reuse that part of code. But I'm fine as it is now.